bciers/apps/reporting/src/data/jsonSchema/commonSchema/common.tsx

```diff
@@ -0,0 +1,14 @@
+import { WidgetProps } from "@rjsf/utils";
```

Comment:
Let's call this file `productionDataTitleWidget.tsx` , it'll be very hard to find otherwise